[INFO -- 2018-11-05 19:52:57]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-05 19:52:57]
Initialization strings:
[INFO -- 2018-11-05 19:52:57]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-05 19:52:57]
(2) set args /tmp/lreg/randInput /tmp/lreg/output /tmp/lreg/detected.log
[INFO -- 2018-11-05 19:52:57]
initSignal:0
[INFO -- 2018-11-05 19:52:57]
endSignal:10
[INFO -- 2018-11-05 19:52:57]
faultModel:0
[EXCEPTION -- 2018-11-05 19:53:03]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-11-05 19:53:03]
gdbException: Cannot access memory at address 0x555d55772380
[INFO -- 2018-11-05 19:53:03]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad88b7 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7adea06 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7adea41 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7adec74 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b06ed2 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x0000555555557fa0 in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd590, __n=18446744069414584324) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x0000555555557ced in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744069414584324) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x000055555555750e in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd590, __n=18446744069414584324) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x0000555555556c24 in std::vector<double, std::allocator<double> >::_M_allocate_and_copy<__gnu_cxx::__normal_iterator<double const*, std::vector<double, std::allocator<double> > > > (this=0x7fffffffd590, __n=18446744069414584324, __first=<error reading variable: Cannot access memory at address 0x555d55772380>, __last=0) at /usr/include/c++/7/bits/stl_vector.h:1260
#11 0x0000555555556523 in std::vector<double, std::allocator<double> >::operator= (this=0x7fffffffd590, __x=std::vector of length -4294967292, capacity -4294967292 = {...}) at /usr/include/c++/7/bits/vector.tcc:206
#12 0x0000555555555691 in train (iterations=45000, alpha=2.2222199999999999e-05, N=4, stopsWhenStable=true) at lreg.cpp:119
#13 0x0000555555555a1c in execute (input=0x7fffffffe1f3 "/tmp/lreg/randInput") at lreg.cpp:157
#14 0x0000555555555c60 in main (argc=4, argv=0x7fffffffde48) at lreg.cpp:191

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Memory content before bitflip:0x040x000x000x000xff0xff0xff0xff
Fault Model: Single bit-flip
base address to flip value: 0x7fffffffd4c0
address max offset: 8
Memory content after  bitflip:0x040x100x000x000xff0xff0xff0xff
frame name: std::vector<double, std::allocator<double> >::_M_allocate_and_copy<__gnu_cxx::__normal_iterator<double const*, std::vector<double, std::allocator<double> > > >
symbol name: __n
symbol filename: /usr/include/c++/7/bits/stl_vector.h
symbol line: 1257
value: 18446744069414588420
value address: 0x7fffffffd4c0
Type: The type is an integer type.
Type sizeof: 8

[INFO -- 2018-11-05 19:53:03]
Fault Injection Successful after 6.7052178382873535s
[INFO -- 2018-11-05 19:53:04]
event type: exit
[EXCEPTION -- 2018-11-05 19:53:04]
exit code: no exit code available
